src/System.ServiceModel.Http/tests/ServiceModel/BasicHttpBindingTest.cs

```diff
@@ -169,6 +169,7 @@ public static void Name_Property_Sets(string value)
     [Theory]
     [InlineData(null)]
     [InlineData("")]
+    [ActiveIssue(1449)]
```

Comment:
When I merge in my big unit test sweep for WcfFact, I'll convert the ActiveIssue to the new Issue form and condition them on FrameworkID.NETNative